recipe-box-frontend/src/main/frontend/src/App.tsx

```diff
@@ -11,6 +12,7 @@ function App() {
             <Routes>
                 <Route path="/" element={<HomePage />} />
                 <Route path="/recipeslist" element={<RecipesList />} />
+                <Route path="/recipeView" element={<RecipeView />}/>
```

Comment:
Add an id parameter for the recipe. More details here:
- URL Parameters: https://v5.reactrouter.com/web/example/url-params
- Request Params: https://medium.com/@bobjunior542/how-to-use-usesearchparams-in-react-router-6-for-url-search-parameters-c35b5d1ac01c

You can also use loaders in order to process the data before processing the page. More details here:
https://reactrouter.com/en/main/route/route
https://stackoverflow.com/questions/78391315/how-to-access-query-parameters-search-params-in-loader-function-in-react-router

This way you can return the mock data from the loader and have the parameter extraction for the RecipeView page prepared for when the api endpoint is ready

Reply:
I have a problem with this review and I can't figure it out. If you could help me with a call tomorrow, I would appreciate it. I’m getting this error: "useLoaderData must be used within a data router. See https://reactrouter.com/routers/picking-a-router."